api/src/main/java/jakarta/data/page/PageRequest.java

```diff
@@ -285,29 +285,29 @@ static <T> Pageable<T> ofSize(int maxPageSize) {
     List<Sort<T>> sorts();
 
     /**
-     * <p>Returns the <code>Pageable</code> requesting the next page
+     * <p>Returns the <code>PageRequest</code> requesting the next page
      * if using offset pagination.</p>
      *
      * <p>If using keyset pagination, traversal of pages must only be done
-     * via the {@link KeysetAwareSlice#nextPageable()},
-     * {@link KeysetAwareSlice#previousPageable()}, or
+     * via the {@link KeysetAwareSlice#nextPageRequest()},
+     * {@link KeysetAwareSlice#previousPageRequest()}, or
      * {@link KeysetAwareSlice#getKeysetCursor(int) keyset cursor},
      * not with this method.</p>
      *
-     * @return The next pageable.
-     * @throws UnsupportedOperationException if this <code>Pageable</code> has a
-     *         {@link Pageable.Cursor Cursor}.
+     * @return The next PageRequest.
```

Comment:
```suggestion
     * @return The next page request.
```
or we could write it as `{@code PageRequest}`. Either would be fine.

Reply:
sorry, I did not see this one, I will create PR to fix it.

Reply:
No problem. In this case, it is just a minor detail of formatting how the text is displayed. It would be fine to skip it.